### PR TITLE
Fix main readme issues to the best of my knowledge

### PR DESCRIPTION
**Pull request by artemkovalyov**
## Context

- [x] Addins SAP scope is not needed anymore?
- [x] VDM installation seemed wrong
- [x] Minor fixes and updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,26 +48,26 @@ To install the SAP Cloud SDK test-util as development dependencies in your proje
 $ npm install -D @sap-cloud-sdk/test-util
 ```
 
-## API documentation
-We use GitHub Pages for our API documentation, the link can be found [here](https://sap.github.io/cloud-sdk/docs/js/api-reference-js-ts).
+## Documentation
+Check out our [API documentation](https://sap.github.io/cloud-sdk/docs/js/api-reference-js-ts)
+and [getting started](https://sap.github.io/cloud-sdk/docs/js/getting-started) guide on [documentation portal](https://sap.github.io/cloud-sdk/)
```
Comment (marikaner) [inline README.md:53]:
```suggestion
Check out our [API documentation](https://sap.github.io/cloud-sdk/docs/js/api-reference-js-ts) and [getting started](https://sap.github.io/cloud-sdk/docs/js/getting-started) guide on our [documentation portal](https://sap.github.io/cloud-sdk/).
```

File: README.md [hunk 2]
```diff
@@ -48,26 +48,26 @@ To install the SAP Cloud SDK test-util as development dependencies in your proje
...
 
 ## Related Projects
 
 ### SAP Cloud SDK CLI
 To reduce the development effort even more, you can use the [SAP Cloud SDK command line interface (CLI)](https://github.com/sap/cloud-sdk-cli) to start a new [NestJS](https://github.com/nestjs/nest) project, including the SDK right from the get go and supporting you do deploy your project to SAP Cloud Platform.
 
 ### Virtual Data Model (VDM)
-In addition to the Open Source parts of this project, we also publish the SAP Cloud SDK Virtual Data Model (VDM) - JavaScript / TypeScript clients for the latest versions of the whitelisted OData services of SAP S/4HANA Cloud under the SAP Developer licence. Those are not included in this project, but rely on modules from this project.
+In addition to the Open Source parts of this project, we also publish the SAP Cloud SDK Virtual Data Model (VDM) - JavaScript / TypeScript pre-generated type-safe clients for the latest versions of the whitelisted OData services of SAP S/4HANA Cloud under the SAP Developer license. Those are not included in this project, but rely on modules from this project.
```
Comment (marikaner) [inline README.md:65]: Good catch on the typo. I find it stumbly to read with the additional adjectives and I think it does not give you more relevant information. Do we have a general section on the VDM in our documentation? I couldn't find it. That could make sense here.
```suggestion
In addition to the Open Source parts of this project, we also publish the SAP Cloud SDK Virtual Data Model (VDM) - JavaScript / TypeScript clients for the latest versions of the whitelisted OData services of SAP S/4HANA Cloud under the SAP Developer license. Those are not included in this project, but rely on modules from this project.
```

**Comment by artemkovalyov** [comment]
I agree on VDM @marikaner , let's leave it as is for now. But in the docs, we decided to switch to the type-safe client as VDM is more of an internal term. It doesn't hurt, for now, so let's keep it easy to read :)

**Comment by marikaner** [comment]
> I agree on VDM @marikaner , let's leave it as is for now. But in the docs, we decided to switch to the type-safe client as VDM is more of an internal term. It doesn't hurt, for now, so let's keep it easy to read :)

Oh yes, I am all for replacing the term VDM completely. No one new can understand this. You need to know it.

File: README.md [hunk 3]
```diff
@@ -48,26 +48,25 @@ To install the SAP Cloud SDK test-util as development dependencies in your proje
...
-In addition to the Open Source parts of this project, we also publish the SAP Cloud SDK Virtual Data Model (VDM) - JavaScript / TypeScript clients for the latest versions of the whitelisted OData services of SAP S/4HANA Cloud under the SAP Developer licence. Those are not included in this project, but rely on modules from this project.
+In addition to the Open Source parts of this project, we also publish the SAP Cloud SDK Virtual Data Model (VDM) - JavaScript / TypeScript clients for the latest versions of the whitelisted OData services of SAP S/4HANA Cloud under the SAP Developer license. Those are not included in this project, but rely on modules from this project.
 
-To install an OData client for an SAP S/4HANA service, make sure you have an `.npmrc` in the root of your project, that contains a reference to the the SAP npm registry: `@sap:registry=https://npm.sap.com`.
+To install an OData client for an SAP S/4HANA service run:
 
-Then run:
 ```sh-session
-$ npm install @sap-cloud-sdk/core
+$ npm install @sap/cloud-sdk-vdm
```
Comment (marikaner) [inline README.md:69]:
```suggestion
$ npm install @sap/cloud-sdk-vdm-<service name>-service
```

File: README.md [hunk 4]
```diff
@@ -48,26 +48,25 @@ To install the SAP Cloud SDK test-util as development dependencies in your proje
...
 ```
```
Comment (marikaner) [inline README.md:71]:
```suggestion
In the example above, `service name` is the name of the service you want to use, e. g. for the business partner service, run: `npm install @sap/cloud-sdk-vdm-business-partner-service`.
```

**Comment by artemkovalyov** [comment]
Thanks for vigilance. Take another look.

**Pull request merged**